webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -94,8 +94,20 @@ const ContentViewVersions = ({ cvId, details }) => {
     } = cvVersion;
     const { task } = activeHistory[0];
     const { result } = task || {};
+
     if (result !== 'error' && !pollIntervals[pollTaskKey(task.id)]) {
-      dispatch(startPollingTask(task.id, task));
+      dispatch(startPollingTask(
+        task.id, task,
+        ({
+          data,
+        }) => {
+          if (!data) return;
+          const { available_actions: { cancellable, resumable } } = data;
```

Comment:
I'm not too sure if checking cancellable and resumable is an accurate reflection of all tasks in general. I think checking for result === error or result === success will make this easier to follow.

Reply:
I was going to say the same thing :smile:

Reply:
> makes sense, thanks @Andrewgdewar!
> 
> As an extra precaution it might be good to also add a cleanup in a useEffect return, so that we cancel any active polling when the component is unmounted. I noticed the errant polling continues even after I switch to a different React page.

Will Do